compiler/q-implant/README.md

```diff
@@ -2,13 +2,21 @@
 
 _q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
 
+Therefore, before proceeding with q-implant, you should define `qparam.json` and `\*.npy` files so that they can be applied to the same operand as the circle you want to apply.
```

Comment:
what is `\` in `*.npy`?

Reply:
It is a mistake that occurred during the local editing process. 
I'll be careful not to make a mistake.